server/routes/follower.js

```diff
@@ -0,0 +1,91 @@
+const { Op } = require("sequelize");
+const express = require("express");
+const router = express.Router();
+const authMiddleware = require("../middlewares/authMiddleware");
+const {
+  followerIdCheck,
+  validate,
+  giveServerError,
+} = require("../middlewares/validate");
+const { UserModel, UserFollowerModel } = require("../models/models");
+const db = require("../models");
+
+//Gets all the users the user is currently following, and from that gets all the users that aren't part of that group.
+const getStrangers = async (req, res) => {
+  let transaction = null;
+  try {
+    transaction = await db.transaction();
+    const followedUsers = await UserModel.findAll({
+      where: {
+        "$FollowedUser.id$": req.user.id,
+      },
+      include: {
+        association: "FollowedUser",
+      },
+      attributes: ["id"],
+      transaction,
+    });
+    const strangers = await UserModel.findAll({
+      where: {
+        id: {
+          [Op.notIn]: followedUsers.map((user) => user.id).concat(req.user.id),
```

Comment:
is `concat` necessary here? `notIn` works on arrays I believe?

Reply:
`concat()` returns an array here. I use it instead of `push()` since that doesn't return an array and I wanted everything in one expression.